tests/test_modelscope.py

```diff
@@ -0,0 +1,26 @@
+import os
+os.environ["GPTQMODEL_USE_MODELSCOPE"] = "True"
+import sys
+import subprocess  # noqa: E402
+import importlib.util  # noqa: E402
+
+if importlib.util.find_spec("modelscope") is None:
+    subprocess.check_call([sys.executable, "-m", "pip", "install", "modelscope", "-U"])
+
+from models.model_test import ModelTest  # noqa: E402
+from gptqmodel import GPTQModel
+
+
+class TestLoadModelscope(ModelTest):
+
+    @classmethod
+    def setUpClass(self):
+        self.MODEL_ID = "Qwen/Qwen2.5-0.5B-Instruct-GPTQ-Int4"
+
+    def test_load_modelscope(self):
+        model = GPTQModel.load(self.MODEL_ID)
+
+        result = model.generate("Hello")[0]
+        assert(model.tokenizer.decode(result))
```

Comment:
Please use a better prompt + assert to make sure the model generates but also generates correctly.

Example: "The capital of France is"
Assert that "paris" is in the `result.lower()`